doc/WebSite/Multi-Lingual-Entities.md

```diff
@@ -0,0 +1,89 @@
+### Introduction
+
+ASP.NET Boilerplate provides two basic interfaces for Multi-Lingual entity definitions. 
```

Comment:
This document should be more clear about why ABP introduces this feature.
We are trying to provide a standard model for translating entities.
I think this document should also say something about "how to update a multi lingual entity (either localize it for a specific language or update the entity completely)".